### DISCUSSION TITLE
Questions about zpwr

### DISCUSSION THREAD

**Discussion by Kritiqual** [MenkeTechnologies/zpwr#16] [Q&A] [answered]
Hi, great work overall. This, however, contains a lot of feature, but I just want to use a small amount of them and organize it in my way. So I have a few questions to ask:

- What is the code responsible for `color-coded groups of zsh menu completion`? Also for `fzf preview pane show contents of JAR, WAR, deb, rpm, zip, tgz and gzip files controlled by central FZF.*OPTS env vars`?
- What are the pros and cons of your `zsh-more-completions` over `zsh-users/zsh-completions`? Can I combine both?

If you don't mind, please keep this issue open (Or how about move this into discussion?)

**Comment by MenkeTechnologies** [comment, marked as answer]
1. The menucompletion color config is in `autoload/common/zpwrBindZstyle`.
You can turn off all colors with
```
export ZPWR_COLORS=false
export ZPWR_COLORS_SECTIONS=false
```
The fzf preview config is in `autoload/common/zpwrBindFzfLate`

Try `'builtin' command rg --hidden --color=always -i -- 'FZF.*OPTS'`

2. `zsh-more-completions` has more completions but most are generated with scripts.  `zsh-users/zsh-completions` is handwritten, higher quality less quantity.  Default config uses both.
